src/lib/services-manager/dialogs/services_manager.rb

```diff
@@ -182,13 +182,18 @@ def services_table
       # @return [Yast::Term]
       def service_buttons
         buttons = [
-          start_stop_button.widget,
-          HSpacing(1),
           start_mode_button.widget,
           HStretch(),
           show_details_button.widget
         ]
 
+        unless Mode.config
```

Comment:
NP: I am fine as it is know, but why not to follow the same mechanism as `show_logs_button?`. I mean, the client decides whether to show that button according to some criteria.